src/flick/show/utils.py

```diff
@@ -33,46 +36,65 @@
 """
 
 
-def get_movie_from_DBinfo(info):
+def get_movie_from_DBinfo(info, credit):
     """
     get a flick movie object similar by parsing the
     information returned by movieDB
     """
     genres = info.get("genres")
-    tags = [genre.get("name") for genre in genres] if genres else []
+    crew = credit.get("crew")
+    cast = credit.get("cast")
+    tags = [g.get("name") for g in genres] if genres else []
+    castInfo = [p.get("name") for p in cast] if cast else []
+    directors = []
+    for c in crew:
+        if c.get("job") == "Director":
+            directors.append(c.get("name"))
 
     movie = {
         "show_id": info.get("id"),
         "title": info.get("original_title"),
-        "poster_pic": info.get("poster_path"),
+        "poster_pic": BASE_MOVIE_URL + info.get("poster_path"),
```

Comment:
oof since I'll be merging my other PR before you, just wait for that and rebase it